index.html

```diff
@@ -6,6 +6,392 @@
+					<figcaption>
+					<h3>Complexity</h3>
+					“Taking 15 medications every day is tough. Especially since they all have different instructions. How am I supposed to keep track of everything?”
+					</figcaption>
+				</figure>
+			</figure>
+		</section>
+	</section>
+	<!-- end why people don't adhere -->
+
+	<!-- The Question -->
+	<section class="row">
+		<span class="subTitle">THE QUESTION</span>
+		<h2 class="sectionDescription">How can technology help patients deal with these factors, so they adhere more accurately and consistently?</h2>
+		<!-- Float both the sub-article and the image left -->
+		<article>
+			<p>
+			Access was our primary concern. Smartphone adoption is growing but 35% of the U.S. population still doesn’t own one.<span>4</span> Smartphones offer many additional features but impose a barrier of entry that seemed too high. Reliance on personal computers is also problematic, especially among disadvantaged patients. 
+			</p>
+			<p>
+			We narrowed our exploration to technology that was pervasive and familiar. Nearly 9 out of 10 American adults use a phone capable of texting.<span>5</span> Furthermore, adherence studies using text reminders show promising results.<span>6</span>  Texting shows the potential to address difficult issues around the world and across verticals. In Libya, the development of the world’s first SMS voter registration system led to 1.5 million voter registrations.<span>7</span>
+			</p>
+			<p>
+			Could texts be used not only to send reminders but also to build an ongoing two-way dialogue with a trusted health partner? Would this conversation help improve medication adherence?
+			</p>
+		</article>
+
+		<!-- rotate the figure using css -->
+		<figure>
+			<img src="assets/phone.svg">
+		</figure>
+	</section>
+	<!-- end the question -->
+
+	<section class="row" id="meetFlorence">
+		<header>
+			<figure>
+				<img src="assets/florence.svg" alt="Florence is a friendly looking nurse with strawberry blonde hair">	
+			</figure>
+			<span class="subTitle">THE CONCEPT</span>
+			<h1>Meet Florence</h1>
+			<p>
+				Our explorations led to the concept of Florence — a personal companion that would help patients understand and manage their medication regimen.  A chatbot at heart, patients would converse with her through text and picture messages. 
+			</p>
+			<p>
+				If developed, a simple, familiar interface would hide her powerful intelligence. Behind the scenes, machine learning and behavioral modeling would enable her to offer a tailored, human experience based on each patient’s needs.
+			</p>
+		</header>
+
+		<!-- nest sections within the meetFlorence section -->
+		<!-- these sections will be styled as white boxes and share a common class -->
+
+		<section class="row underFlorence">
+
+		<!-- the firstHalf and secondHalf classes will both be on the same line, side-by-side -->
+
+			<section class="firstHalf">
+				<h1>SHE GETS TO KNOW YOU.</h1>
```

Comment:
Style this caps in your CSS not HTML. You would use ```text-transform: uppercase``` for this effect. Also this wouldn't be an ```h1```. If I'm reading the headings right, it would probably be an ```h3```.